packages/renderer/src/lib/preferences/PreferencesConnectionActions.svelte

```diff
@@ -56,7 +56,7 @@ async function restartConnectionProvider(
   if (providerConnectionInfo.status === 'started') {
     updateConnectionStatus(provider, providerConnectionInfo, 'restart');
     const loggerHandlerKey = startTask(
-      `Restart ${provider.name} ${providerConnectionInfo.name}`,
+      `Restart ${providerConnectionInfo.name}`,
```

Comment:
If you remove the provider name it means you may see inaccurate data ?

Reply:
Not sure I follow? It used to say `Restart Podman Podman Machine my-machine`, now it says `Restart Podman Machine my-machine`. Even the shorter cases like `Delete MicroShift` or if I can fully control the instance name I find it obvious enough and more natural.

Reply:
I mean, if a user has two instances of Podman it will say Restart Podman Machine my-machine while before you have a way to know it was for 'Podman' Provider or for 'Lima' for example (if machine have the same name)

maybe we need an extra field on task to add this kind of secondary information

Reply:
Ah. I have things uniquely named so length (and not getting the name I picked cut off) has been a bigger issue. How about `Restart ${providerConnectionInfo.name} (${provider.name})`?